### PR TITLE
MEN-6671: Pass the actual keepalive user setting to the HTTP configuration

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: src/common/conf/conf.cpp [hunk 1]
```diff
@@ -261,6 +261,7 @@ expected::ExpectedSize MenderConfig::ProcessCmdlineArgs(
 	http_client_config_.client_cert_path = https_client.certificate;
 	http_client_config_.client_cert_key_path = https_client.key;
 	http_client_config_.skip_verify = skip_verify;
+	http_client_config_.disable_keep_alive = connectivity.disable_keep_alive;
```
Comment (kacf) [inline src/common/conf/conf.cpp:264]: Argh! I was so meticulous when I went through the config. Did I really miss it?? :sob:

**Review by kacf — APPROVED** [review]

File: src/common/conf/conf.cpp (hunk 1 above)
Comment (lluiscampos) [reply src/common/conf/conf.cpp:264]: This is one out of many, your catch ratio was still very high!

Comment (oleorhagen) [reply src/common/conf/conf.cpp:264]: There was also a `ssl_engine` setting missing. Just to throw some more coal on the fire :p

**Review by oleorhagen — APPROVED** [review]

File: src/common/conf/conf.cpp (hunk 1 above)
Comment (kacf) [reply src/common/conf/conf.cpp:264]: Haha, you couldn't pass up that chance, could you! 😃

**Comment by lluiscampos** [comment]
As discussed in person, the user configuration `DisableKeepAlive` refers to HTTP Keep-Alive, not TCP keepalive. This "fix" is therefore unnecessary. I'll follow-up in other PRs

File: src/common/conf/conf.cpp (hunk 1 above)
Comment (oleorhagen) [reply src/common/conf/conf.cpp:264]: hahaha, no :laughing:

**Pull request closed without merging**